### DISCUSSION TITLE
Where can I find a changelog for human?

### DISCUSSION THREAD

**Discussion by dartess** [vercel/next.js#59942] [Help] [answered]
### Summary

https://github.com/vercel/next.js/releases doesn't help me find changes from next@13.3.4 to @13.5.4.

I literally don't understand what could have changed there and how it will affect me.

**Comment by wwwdepot** [comment, marked as answer]
Read their blog https://nextjs.org/blog

**Comment by jvandenaardweg** [comment]
Here you go: https://nextjs-changelog.vercel.app/?version=13.3.4:13.5.4
